lib/galaxy/tool_util/deps/containers.py

```diff
@@ -204,9 +204,9 @@ def __build_container_resolvers(self, app_info):
             log.debug("Unable to find config file '%s'", conf_file)
             return self.__default_containers_resolvers()
         plugin_source = plugin_config.plugin_source_from_path(conf_file)
-        return self.__parse_resolver_conf_xml(plugin_source)
+        return self.__parse_resolver_conf(plugin_source)
```

Comment:
I don't think we need name mangling.

```suggestion
        return self._parse_resolver_conf(plugin_source)
```